le_petit_prince/README.md

```diff
@@ -0,0 +1,61 @@
+## Display settings
+
+* Clone the primary display to the slave monitor.
+* Display resolution - 1920 x 1080, refresh rate - 60 Hz.
+
+## How to run
+
+* Download the audio files (`.wav` files with suffix `task-lppFR`) from OpenNeuro: https://openneuro.org/datasets/ds003643/versions/2.0.1
```

Comment:
I have added the openneuro link in the README. We should probably add more info about where to save these stim files so that the script can find it.

Reply:
> I had added the openneuro link in the README. We should probably add more info about where to save these stim files so that the script can find it.

Maybe in that case having the files directly here is better, so that we don't give much room for confusion?

Reply:
We definitely can put the files here, Christophe also said so. We were discussing simply pointing to openneuro, so as to avoid any legal issues and having to take them down from multiple places in the future.

Reply:
I may have said the converse recently, But I think that  it's OK to just have the link.